package.json

```diff
@@ -5,6 +5,7 @@
   "main": "lib/index.js",
   "types": "index.d.ts",
   "dependencies": {
+    "@types/express-jwt": "0.0.41",
```

Comment:
this should be a dev dependency

Reply:
The TypeScript handbook recommends using `dependencies`, because otherwise consumers will have to install `@types/express-jwt` manually: https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#dependencies

Reply:
Thanks for the link!